### PR TITLE
System test cleanup

### PR DESCRIPTION
**Pull request by edsantiago**
- cp test: clean up stray image

- build test: add workaround for #9567 (ultra-slow ubuntu).
  We're seeing CI flakes (timeouts) due to ubuntu 2004 being
  absurdly slow. Workaround: double our timeout on one specific
  test when ubuntu + remote.

- build test: clean up new copy-from test (from #9275).
  The test was copy-pasted from buildah system tests, without
  really adapting for podman environment (e.g. it was using
  images that we don't use here, and would cause pulls, which
  will cause flakes). Rewrite test so it references only $IMAGE,
  remove some confusing/unnecessary stuff, selectively run
  parts of it even when rootless or remote, and add a
  test to confirm that copy-from succeeded.

- load test: add error-message test to new load-invalid (#9672).
  Basically, make sure the command fails for the right reason.

- play test (kube): use $IMAGE, not alpine; and add pause-image
  cleanup to teardown()

- apiv2 mounts test: add a maintainability comment in a tricky
  section of code; and tighten up the mount point test.

Signed-off-by: Ed Santiago <santiago@redhat.com>

### CODE REVIEW

**Comment by edsantiago** [comment]
Rebased to incorporate time zone fix from #9716

**Comment by TomSweeneyRedHat** [comment]
LGTM

File: test/system/065-cp.bats [hunk 1]
```diff
@@ -370,6 +370,7 @@ load helpers
     is "${lines[0]}" "${randomcontent[0]}" "eval symlink - created container"
     is "${lines[1]}" "${randomcontent[1]}" "eval symlink - created container"
     run_podman rm -f cpcontainer
+    run_podman rmi $cpimage
```
Comment (vrothberg) [inline test/system/065-cp.bats:373]: Thanks for cleaning up :)
